platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/EntityController.java

```diff
@@ -70,7 +70,7 @@
                     byModule.put(entity.getModule(), new ArrayList<EntityDto>());
                 }
 
-                if (!entity.isAbstractClass()) {
+                if (!entity.isAbstractClass() || entity.isDiscriminated()) {
```

Comment:
So why are we displaying these? Are we displaying all instances of entities inheriting this class?